.github/workflows/build.yml

```diff
@@ -28,6 +28,13 @@ jobs:
           ${{format(matrix.generator != 'Default Generator' && '-G "{0}"' || '', matrix.generator)}}
       runs-on: ${{ matrix.os }}
       steps:
+
+        - uses: awalsh128/cache-apt-pkgs-action@latest
```

Comment:
This seems like a random action; is there no standard form? Or maybe a way to just install the dependency in this action?

Also, this looks like you're just adding the installation to the main flow, rather than adding another test. Can you add a x-product in the matrix instead?

Reply:
please help me, I don't really understand yaml

Reply:
This is more of a GitHub actions syntax thing. I think there's documentation about it on the Internet.

Reply:
ok changed a little bit the matrix